awx/main/managers.py

```diff
@@ -22,10 +22,7 @@ class HostManager(models.Manager):
 
     def active_count(self):
         """Return count of active, unique hosts for licensing."""
-        try:
-            return self.order_by('name').distinct('name').count()
-        except NotImplementedError: # For unit tests only, SQLite doesn't support distinct('name')
-            return len(set(self.values_list('name', flat=True)))
+        return self.values('name').distinct().count()
```

Comment:
Did Django+sqlite grow support for `.distinct()` ?

Reply:
It could be from the Django upgrade? I put in those unit tests, and they run using the same query, and they're correct. There might be some deep-level gotchas here, but I'm not seeing them.

Reply:
but it wasn't really that it didn't have support for .distinct(), if it didn't have that, _way_ more stuff would have been broken in the unit tests. It's some complicated "distinct on" logic, I know the condition is complicated, but don't pretend to know much more than that.